src/app/components/Menu/Menu.tsx

```diff
@@ -294,27 +343,54 @@ const Menu = () => {
               {showLanguages && (
                 <div className="languagesListContainer--mob">
                   <ul className="languagesList">
-                    <button
-                      type="button"
-                      className="languagesList__button"
-                      onClick={handleLanguageSelection}
-                    >
-                      Polish
-                    </button>
-                    <button
-                      type="button"
-                      className="languagesList__button"
-                      onClick={handleLanguageSelection}
-                    >
-                      Spanish
-                    </button>
-                    <button
-                      type="button"
-                      className="languagesList__button"
-                      onClick={handleLanguageSelection}
-                    >
-                      French
-                    </button>
+                    {actualLanguage !== 'English' && (
```

Comment:
Czemu my to tataj powtarzamy znowu? Zrob z tego jeden komponent bo logika jest ta sama. Wyglad mozesz zmieniac za pomocą jakiegos propsa czy cos